atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1773,6 +1774,27 @@ private void alterGcAndTombstoneWithLock(
         }
     }
 
+    public void cleanUpSchemaMutationLockTablesState() throws Exception {
+        // get a unique table and create it if it doesn't exist
+        Set<TableReference> tables = lockTables.getAllLockTables();
+        TableReference unique;
+        if (tables.size() == 0) {
+            unique = lockTables.createLockTable();
+            log.info("There were no locks tables, so one was succesfully created: {}", unique.toString());
+            return;
+        }
+        unique = tables.iterator().next();
+        tables.remove(unique);
```

Comment:
Cool, I'm happy to just make it drop everything, and I can bake a re-name refactor into this.  Your understanding of this is correct and waitForSchemaVersions will make sure the change propagates.